lib/handlers/quote/injector.ts

```diff
@@ -135,6 +139,10 @@ export class QuoteHandlerInjector extends InjectorSOR<
           tokenValidatorProvider,
           tokenPropertiesProvider,
           v2Supported,
+          v4Supported,
```

Comment:
i forgot to pass in those params, but those are either v4 or mixed related anyway. for mixed it's a no-op, we only deployed MixedRouteQuoterV1 to L1 mainnet and base for production nets, so we cannot enable mixed to any other chains.

Next week, I'm planning to deploy MixedRouteQuoterV2 (and maybe MixedRouteQuoterV1 to the rest of the chains) myself, and then we can gradually roll out mixed route support per chain with percent control in https://github.com/Uniswap/routing-api/blob/main/lib/handlers/quote/util/quote-provider-traffic-switch-configuration.ts#L11